src/components/ProgressBar/ProgressBar.jsx

```diff
@@ -0,0 +1,91 @@
+import PropTypes from 'prop-types'
+import React, { useMemo } from 'react'
+import styled, { css } from '@xstyled/styled-components'
+
+import { Flex, Caption } from '../'
+
+import SvgCheckCircle from '../../icons/CheckCircle'
+import SvgError from '../../icons/Error'
+
+const ProgressBar = ({ size, type, successColor, loadingColor, errorColor, progress, error }) => {
+  const barColor = useMemo(() => {
+    if (progress === 100) {
+      return successColor
+    }
+
+    if (error === true) {
```

Comment:
não precisa usar `=== true` :)

```suggestion
    if (error) {
```